### PR TITLE
QAT/NGINX changes for Release v21.6

### PR DESCRIPTION
**Pull request by dpatel257**

### CODE REVIEW

**Comment by jhou5** [comment]
dev image is sw config, should include(ipsecmb.m4), i.e.
https://github.com/OpenVisualCloud/Dockerfiles/blob/4a8143cf48dd19680e9b796f8b20a6a5c600cc44/QAT/ubuntu-20.04/media/dev/Dockerfile.m4?

**Comment by dpatel257** [comment]
> dev image is sw config, should include(ipsecmb.m4), i.e.
> https://github.com/OpenVisualCloud/Dockerfiles/blob/4a8143cf48dd19680e9b796f8b20a6a5c600cc44/QAT/ubuntu-20.04/media/dev/Dockerfile.m4?

yes, missed out on that, thanks, will fix it

**Comment by dpatel257** [comment]
> > dev image is sw config, should include(ipsecmb.m4), i.e.
> > https://github.com/OpenVisualCloud/Dockerfiles/blob/4a8143cf48dd19680e9b796f8b20a6a5c600cc44/QAT/ubuntu-20.04/media/dev/Dockerfile.m4?
> 
> yes, missed out on that, thanks, will fix it

Can you check now?

File: QAT/centos-7/media/nginx/Dockerfile [hunk 1]
```diff
@@ -98,29 +98,16 @@ RUN cd /opt/build && \
...
 # load qat-engine
-ARG QAT_ENGINE_REPO=https://github.com/intel/QAT_Engine/archive/v0.6.4.tar.gz
+ARG QAT_ENGINE_REPO=https://github.com/intel/QAT_Engine/archive/v0.6.6.tar.gz
 RUN cd /opt/build && \
     wget -O - ${QAT_ENGINE_REPO} | tar xz && \
     cd QAT_Engine* && \
     ./autogen.sh && \
     export PERL5LIB="$(ls -1 -d /opt/build/openssl-*)" && \
-    ./configure --with-qat_dir=/opt/intel/QAT --with-openssl_dir="$PERL5LIB" --with-openssl_install_dir=/usr/local/ssl --prefix=/opt/intel/QATengine --disable-qat_ecx --with-cc-opt="-DQAT_DISABLE_NONZERO_MEMFREE" --enable-multibuff_offload --enable-multibuff_ecx && \
+    ./configure --with-qat_dir=/opt/intel/QAT --with-openssl_dir="$PERL5LIB" --with-openssl_install_dir=/usr/local/ssl --prefix=/opt/intel/QATengine --disable-qat_ecx --with-cc-opt="-DQAT_DISABLE_NONZERO_MEMFREE -IBUILD_HOME/ipp-crypto-QAT_CRYPTOMB_VER/sources/ippcp/crypto_mb/include" --enable-qat_hw --with-qat_hw_dir=/opt/intel/QAT && \
```
Comment (jhou5) [inline QAT/centos-7/media/nginx/Dockerfile:110]: -IBUILD_HOME/ipp-crypto-QAT_CRYPTOMB_VER/sources/ippcp/crypto_mb/include" should not be there as ipp-crypto removed

**Review by jhou5 — APPROVED** [review]

**Pull request merged**